WhaleSpotting/Services/SightingsService.cs

```diff
@@ -75,11 +76,28 @@ public async Task<SightingResponseModel> ConfirmSighting(int id)
             {
                 return null;
             }
-            
+
             sighting.Confirmed = true;
             _context.SaveChanges();
 
             return new SightingResponseModel(sighting);
         }
+
+        public async Task<IEnumerable<string>> GetSpeciesByCoordinates(string latitude, string longitude)
+        {
+            var fiftyKmInCoords = 0.45;
+            var upperLatitude = double.Parse(latitude) + fiftyKmInCoords;
```

Comment:
With the change to the endpoint we can remove the double.Parse for all of these

Reply:
Addressed.